MANUAL.txt

```diff
@@ -1381,14 +1381,48 @@ Metadata variables
         ...
 
 `subtitle`
-:   document subtitle, included in HTML, EPUB, LaTeX, ConTeXt, and Word docx
+:   document subtitle, included in HTML, EPUB, LaTeX, ConTeXt, and docx
+    documents
 
 `abstract`
-:   document summary, included in LaTeX, ConTeXt, AsciiDoc, and Word docx
+:   document summary, included in LaTeX, ConTeXt, AsciiDoc, and docx
+    documents
 
 `keywords`
-:   list of keywords to be included in HTML, PDF, and AsciiDoc metadata;
-    repeat as for `author`, above
+:   list of keywords to be included in HTML, PDF, ODT, pptx, docx
+    and AsciiDoc metadata; repeat as for `author`, above
+
+`subject`
+:   document subject, included in ODT, docx and pptx metadata
+
+`description`
+:   document description, included in ODT, docx and pptx metadata. Some
+    applications show this as `Comments` metadata.
+
+`category`
+:   document category, included in docx and pptx metadata
+
+`custom-properties`
```

Comment:
It's potentially a bit misleading to have `custom-properties` in this list as if it's also a variable or metadata field.
I'd suggest putting this statement about custom properties outside the list of variables.